guide/private-key-management/cloud-backup.md

```diff
@@ -33,14 +33,16 @@ https://www.figma.com/community/file/995256542920917246/BDG---Private-key-manage
    layout = "full-width"
 %}
 
-# Automatic cloud backup
+# Cloud backup
 
 By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new [bitcoin wallet]({{ '/guide/glossary/wallet/#bitcoin-wallet' | relative_url }}). This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) that is used to initiate a wallet.
```

Comment:
```suggestion
By using an cloud backup for the private key the user is NOT required to manually backup anything when setting up a new [bitcoin wallet]({{ '/guide/glossary/wallet/#bitcoin-wallet' | relative_url }}). This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) that is used to initiate a wallet.
```

If we are changing this page title, like the cloud backups page in onboarding, we should be dropping the 'automatic' term for reasons outlined when adding LN content to that section.

Reply:
I'm against dropping 'Automatic' as it's a defining feature of this scheme.

Reply:
I'm against dropping 'Automatic', it's a defining feature of this scheme.

Reply:
I like the idea of a more generic on-device, and cloud backup that can be used for various types of Bitcoin application data and style procedures (manual , automatic).